### PR TITLE
docs: improve README

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**
fix https://github.com/googleapis/synthtool/issues/450

and update instructions to match current process

### CODE REVIEW

File: synthtool/README.md [hunk 1]
```diff
@@ -13,26 +13,23 @@ This tool helps to generate and layout cloud client libraries. Synthtool runs th
 
 3. **Docker**  Some synth.py files require [Docker] to generate code.
 
+4. Add synthtool to your **PYTHONPATH**
+
+    ```
+    export PYTHONPATH=`realpath .`
```
Comment (danoscarmike) [inline synthtool/README.md:19]: Does `realpath` work on all platforms?

Comment (SurferJeffAtGoogle) [reply synthtool/README.md]: clarified.

File: synthtool/README.md [hunk 2]
```diff
@@ -13,26 +13,24 @@ This tool helps to generate and layout cloud client libraries. Synthtool runs th
 
 3. **Docker**  Some synth.py files require [Docker] to generate code.
 
+4. Add this repo to your **PYTHONPATH**.
+
+    ```
+    cd synthtool
+    export PYTHONPATH=`pwd`
```
Comment (chingor13) [inline synthtool/README.md:20]: Is this really how we suggest people install it locally? Personally, I `python3 -m pip install -e .`

Comment (danoscarmike) [reply synthtool/README.md:20]: That's good point. Unless we recommend avoiding a global package install? Could use `venv` of course.

Comment (SurferJeffAtGoogle) [reply synthtool/README.md]: Done.

**Pull request merged**